chore: update gov with tsc menbers (#964)
docarray/docarray: GOVERNANCE.md

* Submit your code as a PR with the appropriate DCO signoff
* Have your submission approved by the committer(s) and merged into the codebase.

## Core developers

The core developers role enables the contributor to commit code directly to the repository, but also comes with the responsibility of being a responsible leader in the community.

Currently core developers of DocArray are: [Han](https://github.com/hanxiao), [samsja](https://github.com/samsja) , [JoanFM](https://github.com/JoanFM), [JohannesMessner](https://github.com/JohannesMessner), [nan](https://github.com/nan-wang), [alaeddine-13](https://github.com/alaeddine-13), [AnneYang720](https://github.com/AnneYang720), [anna-charlotte](https://github.com/anna-charlotte)

### Process for becoming a core developer

Contributors can become core developers by beeing nominated by at least one other core developers of DocArray. There will be a vote by the current core developers. While it is expected that most votes will be unanimous, a two-thirds majority of the cast votes is enough.

If you want to become a core developers, reach out to docarray@jina.ai. 

## TSC members

> The Techincal Steering Commite (TSC) members are core developers who have additional responsibilities to ensure the smooth running of the project. TSC members are expected to participate in strategic planning, and approve changes to the governance model. The purpose of the TC is to ensure a smooth progress from the big-picture perspective.

Reviewer: ```suggestion
The Techincal Steering Commite (TSC) members are core developers who have additional responsibilities to ensure the smooth running of the project. TSC members are expected to participate in strategic planning, and approve changes to the governance model. The purpose of the TSC is to ensure a smooth progress from the big-picture perspective.
```

Currently TSC menbers of DocArray are: [Han](https://github.com/hanxiao), [samsja](https://github.com/samsja) , [JoanFM](https://github.com/JoanFM), [JohannesMessner](https://github.com/JohannesMessner), [nan](https://github.com/nan-wang), [alaeddine-13](https://github.com/alaeddine-13)

> One of the TSC menber is the chairperson of the TSC that should ensurethe smooth running of the TSC. It does not have more voting power that other TSC menber.

Reviewer: ```suggestion
One of the TSC members is the chairperson of the TSC and should ensure the smooth running of the TSC. They do not have more voting power that other TSC menber.
```

Currently [JoanFM](https://github.com/JoanFM) is the chairperson of the TSC

### Process for becoming a TSC member

> Core developers can become TSC menbers by beeing nominated by at least one TSC menbers, There will be a vote by the current TSC menbers and the vote need to be unanimous between for the nominated person to be accepted as a TSC menbers. TSC members who do not actively engage with the TC duties are expected to resign.

Reviewer: ```suggestion
Core developers can become TSC menbers by beeing nominated by at least one TSC member. There will be a vote by the current TSC menbers and the vote need to be unanimous for the nominated person to be accepted as a TSC menber. TSC members who do not actively engage with the TSC duties are expected to resign.
```
